Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
@weaverryan See the changes in the `unserialize` method below: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/1956/files#diff-a1821a37b4b2e9240f70068f6bdea09eR169